src/main/java/addon/Storage.java

```diff
@@ -0,0 +1,123 @@
+package addon;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.time.LocalDateTime;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import addon.Tasklist.Deadline;
+import addon.Tasklist.Event;
+import addon.Tasklist.Task;
+import addon.Tasklist.Todo;
+
+/**
+ * Handles storing and loading the save file.
+ */
+public class Storage {
+
+    /**
+     * Returns a new list of the current tasks.
+     *
+     * @param list List to be written over.
+     */
+    public static void rewriteFile(ArrayList<Task> list) {
+        File oldFile = new File("./save.txt");
```

Comment:
Might be good to allow users/developers to select their own filepath as an input